docs/_summary.md

```diff
@@ -54,9 +54,12 @@
   * [Bootmagic](feature_bootmagic.md)
   * [Combos](feature_combo)
   * [Command](feature_command.md)
+  * [Debounce API](feature_debuonce.md)
   * [Dynamic Macros](feature_dynamic_macros.md)
   * [Encoders](feature_encoders.md)
   * [Grave Escape](feature_grave_esc.md)
+  * [Haptic Feedback](feature_haptic_feedback.md)
+  * [HD44780 LED](feature_hd44780.md)
```

Comment:
`features.md` calls this `HD44780 LCD Display` rather than LED. I think the most accurate name would be `HD44780 LCD Controller` for both documents. Perhaps leave off the word "Controller" if visual length is an issue.

```suggestion
  * [HD44780 LCD Controller](feature_hd44780.md)
```

## Also

Line 69: the OLED driver link needs a `.md` inserted.